### ISSUE TITLE
CLI removes tns_modules in platforms

### ISSUE THREAD

**Issue by Pip3r4o** [NativeScript/nativescript-dev-webpack#5]
After the prepare-hook is executed the tns_modules folder is completely wiped from `platforms/../src/main/assets/app` where non-js files, xmls, and image resources may reside.

@blagoev @jasssonpet

**Comment by Pip3r4o** [comment]
We can consider removing all .js files in tns_modules thus leaving just the resources that may be referred runtime, and also leave `bundle.js`, and `tns-java-classes` in addition when building for android.

**Comment by hdeshev** [comment]
An alternative solution to this problem would be to provide webpack loaders for known resource types, so that those get bundled as well.

What's the specific case here? Do we have a sample app that needs to reference something from `tns_modules`?

**Comment by gsaadeh** [comment]
I am running into what I believe is a similar issue. When I run the app without the bundle it works just fine. After adding webpack and running the app, I get the following error:

```
Could not find module 'data/observable'. Computed path '/Users/George/Library/Developer/CoreSimulator/Devices/FFF2D358-AC9C-46B1-8200-8E367588D9A3/data/Containers/Bundle/Application/9CB23FDF-570A-4646-BB92-72F09B5FB50D/sample.app/app/tns_modules/data/observable'
```
P.S: I am not using angular2

**Comment by markosko** [comment]
I have same either i got error on `data/observable` or `application` missing in tns_modules folder even on newly created app and bundled i get error but not sure if this is same issue or its something as for new issue

**Comment by gsaadeh** [comment]
I believe it's the same error since I saw the same application error that you mentioned as well. I did a rm -rf and then tns install but didn't solve it.

Sent from my iPhone

On Sep 22, 2016, at 2:48 PM, markosko <notifications@github.com<mailto:notifications@github.com>> wrote:


I have same either i got error on data/observable or application missing in tns_modules folder even on newly created app and bundled i get error but not sure if this is same issue or its something as for new issue

-

**Comment by gsaadeh** [comment]
I believe it's the same error since I saw the same application error that you mentioned as well. I did a `rm -rf` and then `tns install` but nothing helped.

**Comment by hdeshev** [comment]
This should be resolved with nativescript-dev-webpack@0.2.x and nativescript-cli@2.4.x

**Comment by gsaadeh** [comment]
Thanks @hdeshev I'll give it a try
